docs/repository-structure.md

```diff
@@ -0,0 +1,39 @@
+# Repository Structure Overview
+
+* `build` - Helper JS scripts used by `run` script in the main directory.  
+* `docs` - Documentation.
+* `src/js` - the JS part of IDE application.
+* `src/rust` - All Rust crates.
+
+## Crates
+
+The crates structure is currently in process of refactoring, and there are still places where
+code does not fit the crate it is placed. The work is tracked in #448 epic.
```

Comment:
It should be made clear if this document is about the "current" state or about the "desired" state.
Also, I'd be nice to shortly describe the difference between the two.

Reply:
But the work is very dynamic. I rather assumed, that the user will see an epic summary and the progress of tasks of those epic.

It's quite obvious, that non-closed epic describes the _desired_ state, isn't it?